### PR TITLE
[tr] Add Turkish translation for _index.md

### PR DESCRIPTION
**Pull request by rwxdash**
### Describe your changes

- Added `content/tr` directory for Turkish translations
- Added Turkish translation for `_index.md`

Google Docs for reference

- [`_index.md`](https://docs.google.com/document/d/1X0Rph6-CZYSE27qRzpi9_L2uZIa9JespFZ0BG5SV9W4/edit?usp=sharing)

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/tr/_index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Katkıda bulunmak isteyen herkes, bir GitHub issue veya pull request oluşturabilir. Lütfen [Stil Kılavuzu](https://github.com/cncf/glossary/blob/main/style-guide)'nu inceleyip takip ettiğinizden, [Nasıl Katkıda Bulunabilirim?](https://github.com/cncf/glossary/blob/main/contribute) dokümanını okuduğunuzdan, [CNCF'in Slack topluluğuna](https://communityinviter.com/apps/cloud-native/cncf) katıldığınızdan ve [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) kanalına dahil olduğunuzdan emin olun. Ayrıca Cloud Native Sözlüğü'nü kendi ana diline çevirmek isteyenler için [#glossary-localizations](https://cloud-native.slack.com/archives/C02N2RGFXDF) isimli bir kanal da bulunmaktadır. Türkçe çeviriye katkıda bulunmak için [#glossary-localizations-turkish](https://cloud-native.slack.com/archives/C05ATBGLHJ7) kanalına dahil olabilirsiniz.
```
Comment (aliok) [inline content/tr/_index.md:14]: We should update these links later, when we have the localizations of these pages.

Don't worry about it for now though.

File: content/tr/_index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Cloud Native Sözlüğü"
+status: Completed
+---
+
+**Cloud Native Sözlüğü**
```
Comment (aliok) [inline content/tr/_index.md:6]:
```suggestion
# Cloud Native Sözlüğü
```

We should better use the same `#` levels for the headers with the English one. https://raw.githubusercontent.com/cncf/glossary/8df5004a00fc1a78ad2914665e691cfee22a6a47/content/en/_index.md


https://gdoc2md.com/ is producing things a bit different.

File: content/tr/_index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Cloud Native Sözlüğü, karmaşıklığı ile ünlü olan cloud native alanının yalnızca teknoloji uzmanları için değil, herkes için anlaşılabilir olmasını hedeflemektedir. Basitliğe odaklanan sözlük, CNCF Business Value Subcommittee (BVS) tarafından yönetilen bir projedir.
+
```
Comment (aliok) [inline content/tr/_index.md:9]:
```suggestion
Cloud Native Sözlüğü, karmaşıklığı ile ünlü olan cloud native alanının yalnızca teknoloji uzmanları için değil, herkes için anlaşılabilir olmasını hedeflemektedir. Basitliğe odaklanan sözlük, CNCF Business Value Subcommittee (BVS) tarafından yönetilen bir projedir.

<p><img class="mt-3" src="/images/homepage/kubecon.jpg" alt="Kubecon sunumu izleyen insanlar"></p>
```

We missed this image in the Google Doc (which is fine).

In general, we should compare the page with the English version (https://raw.githubusercontent.com/cncf/glossary/8df5004a00fc1a78ad2914665e691cfee22a6a47/content/en/_index.md)

**Review by aliok — COMMENTED** [review]
Added a few comments.

We can use the preview feature of Netlify to compare things visually (EN vs TR): https://deploy-preview-2239--cncfglossary.netlify.app/tr/ vs https://deploy-preview-2239--cncfglossary.netlify.app/

File: content/tr/_index.md (hunk 1 above)
Comment (rwxdash) [reply content/tr/_index.md:14]: Let me make a note in our index sheet for these. Thanks.

File: content/tr/_index.md (hunk 3 above)
Comment (rwxdash) [reply content/tr/_index.md:9]: Thank you.

File: content/tr/_index.md (hunk 2 above)
Comment (rwxdash) [reply content/tr/_index.md:6]: Ah, yes. Thanks. I'll update the rest as well.

**Comment by rwxdash** [comment]
> Added a few comments.
> 
> We can use the preview feature of Netlify to compare things visually (EN vs TR): https://deploy-preview-2239--cncfglossary.netlify.app/tr/ vs https://deploy-preview-2239--cncfglossary.netlify.app/

Mentioned issues and some others are addressed.

**Review by aliok — APPROVED** [review]
Thanks for addressing all comments.

LGTM

**Pull request merged**